### DISCUSSION TITLE
v1.105.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#9486] [Announcements]
# v1.105.0

> [!CAUTION]
> # Changes in glob path external library
> Library import paths no longer support wildcards (* notation/globs). If your library was previously using this, please update affected paths to point to directories instead.
> Note: exclusion paths remain unchanged and still support glob syntax.

## Highlights

Welcome to release `v1.105.0` of Immich. **WOW, 2 releases in 2 days! Immich's full-time core team has taken effect**. Slightly kidding there, but we have a lot of bug fixes since the last release that we would like to push out to avoid you all encountering annoying bugs when using the software. Some highlights can be found below:

- The mobile app now uses a more efficient sync algorithm.
- Fixed assets status not showing correctly when deleting assets from the server when opening the mobile app.

### ⚠️ Breaking Changes
* fix(server): support special characters in library paths by @sushain97 in https://github.com/immich-app/immich/pull/9385
### 🗄️ Server
* fix(server): regenerate (extract) motion videos by @jrasm91 in https://github.com/immich-app/immich/pull/9438
* chore(server): remove unused column by @jrasm91 in https://github.com/immich-app/immich/pull/9431
* fix(server): addAssets and removeAssets handle duplicate assetIds by @eric-barch in https://github.com/immich-app/immich/pull/9436
* refactor(server): logger by @jrasm91 in https://github.com/immich-app/immich/pull/9472
* feat(server): run microservices in worker thread by @zackpollard in https://github.com/immich-app/immich/pull/9426
* fix(server): new full sync return stacked assets individually by @fyfrey in https://github.com/immich-app/immich/pull/9189
* fix(server): sync issue when delete remotes assets by @alextran1502 in https://github.com/immich-app/immich/pull/9479
### 📱 Mobile
* feat(mobile): use efficient sync by @fyfrey in https://github.com/immich-app/immich/pull/8842
### 🖥️ Web
* fix(web): show w x h correctly when media is rotated by @jrasm91 in https://github.com/immich-app/immich/pull/9435
* feat(web): combine auth settings by @jrasm91 in https://github.com/immich-app/immich/pull/9427
* Fix typo in error page title by @donebydavid in https://github.com/immich-app/immich/pull/9451
* fix(web): decrease asset viewer navigation area size by @Snowknight26 in https://github.com/immich-app/immich/pull/9455
* fix(web): admin settings number input validation by @michelheusschen in https://github.com/immich-app/immich/pull/9470
### 📓 Documentation
* fix install.sh add random password for .env by @xiagw in https://github.com/immich-app/immich/pull/9282
* docs: remove mention of delete being disabled for external assets by @benmccann in https://github.com/immich-app/immich/pull/9465
* docs(server,ml): update nvidia toolkit link by @mertalev in https://github.com/immich-app/immich/pull/9477
* Added Swedish translation of README. by @klahr in https://github.com/immich-app/immich/pull/9464
### 🔨 Maintenance
* chore(deps): update node.js to 291e84d by @renovate in https://github.com/immich-app/immich/pull/9449
* fix(deps): update typescript-projects by @renovate in https://github.com/immich-app/immich/pull/9454
* chore(deps): update node.js to 53108f6 by @renovate in https://github.com/immich-app/immich/pull/9450
* chore(deps): update base-image to v20240514 (major) by @renovate in https://github.com/immich-app/immich/pull/9469
* fix(deps): update machine-learning by @renovate in https://github.com/immich-app/immich/pull/9304
* chore(deps): update machine-learning by @renovate in https://github.com/immich-app/immich/pull/9478
* chore(deps-dev): bump werkzeug from 3.0.1 to 3.0.3 in /machine-learning by @dependabot in https://github.com/immich-app/immich/pull/9480
* chore(deps): bump tqdm from 4.66.1 to 4.66.3 in /machine-learning by @dependabot in https://github.com/immich-app/immich/pull/9481
* fix(deps): update dependency @zoom-image/svelte to v0.2.11 by @renovate in https://github.com/immich-app/immich/pull/9482

## New Contributors
* @donebydavid made their first contribution in https://github.com/immich-app/immich/pull/9451
* @eric-barch made their first contribution in https://github.com/immich-app/immich/pull/9436
* @xiagw made their first contribution in https://github.com/immich-app/immich/pull/9282
* @klahr made their first contribution in https://github.com/immich-app/immich/pull/9464

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.104.0...v1.105.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.105.0'>v1.105.0</a>.</em>

**Comment by joelis10** [comment]
I'm liking the new format for the patch notes but I slightly miss the "I'd pay to see that" gif

**Comment by Solarunit** [comment]
I just saw this video "Immich Q&A ft. Eron Wolf & Louis Rossmann from Futo" https://www.youtube.com/watch?v=cwz2iZwYpgg
and have 2 questions:
1. Should we expect analytics/telemetry in immich soon?
2. If yes, will analytics/telemetry be enabled or disabled by default?

**Reply by alextran1502** [reply]
1. Probably in a couple months, we don't have a concrete timeline for this yet.

2. I am leaning toward disable by default.

**Reply by Solarunit** [reply]
Will analytics/telemetry be tightly integrated into core container or it will be in shape of separate container that you can easily remove form yaml file?

**Reply by zackpollard** [reply]
> Will analytics/telemetry be tightly integrated into core container or it will be in shape of separate container that you can remove form yaml file? 

It'd be integrated into the core container, but we also don't have firm details on anything at the moment, we haven't made these decisions.

**Comment by diyoyo** [comment]
Is it safe to upgrade from 103.1 to 105.1 directly?
I have no wildcard anywhere.
Thanks.

**Reply by zackpollard** [reply]
Yes, in general just read the breaking changes for each release (if any) and make sure to apply them all

**Reply by diyoyo** [reply]
Thanks

**Comment by pascoalkill** [comment]
Gostaria de atualizar minha versão do servidor, a atual é 1.94.1

Alguém pode me ajudar como fazer isso?

Nunca atualizei, não gostaria de crachar o server.

Obrigado.

**Reply by j-seixas** [reply]
1. Ir ao changelog (https://github.com/immich-app/immich/releases) 
2. Procurar a sua versão e a seguinte
3. Ver quais as mudanças feitas e se ha breaking changes. Se houver, veja que mudanças ou coisas que precisa de ter em atenção antes do update. Faça o que é recomendado por la.
4. Atualize para a versao vista acima no ponto 2-3
5. Verifique que tudo funciona direito
6. Voltar ao passo 2 ate estar na versao mais recente

Procure ajuda nas discussions para cada versao ou crie nova se encontrar problemas
